packages/openapi3/test/examples.test.ts

```diff
@@ -307,4 +307,47 @@ worksFor(["3.0.0", "3.1.0"], ({ openApiFor }) => {
       },
     });
   });
+
+  it("supports encoding", async () => {
```

Comment:
should have a test also that test that by default headers are encoded as rfc7231